class-tgm-plugin-activation.php

```diff
@@ -451,6 +451,8 @@ public function init() {
 			if ( true === $this->has_forced_deactivation ) {
 				add_action( 'switch_theme', array( $this, 'force_deactivation' ) );
 			}
+
+			add_action( 'admin_head',  array( $this, 'tgm_css' ), 21 );
```

Comment:
Where has this magic `21` come from? Maybe a comment is needed to explain why that priority was chosen.